README.md

```diff
@@ -57,137 +59,68 @@ import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
-
-```ts
-import { configureLoadStyles } from '@microsoft/load-themed-styles';
-
-// Store registered styles in a variable used later for injection.
-let _allStyles = '';
-
-// Push styles into variables for injecting later.
-configureLoadStyles((styles: string) => {
-  _allStyles += styles;
-});
-
-import * as React from 'react';
-import * as ReactDOMServer from 'react-dom/server';
-import { Button } from 'office-ui-fabric-react/lib/Button';
-
-let body = ReactDOMServer.renderToString(<Button>hello</Button>);
-
-console.log(
-  `
-  <html>
-  <head>
-    <style>${_allStyles}</style>
-  </head>
-  <body>
-    ${body}
-  </body>
-  </html>
-  `
-);
-```
-
-Note: We are evaluating a more robust theming and style loading approach, which will allow a much more flexible server rendering approach, so this syntax may be simplified in the future.
-
-### Browserless Testing
-
-In unit or end-to-end tests that run in an SSR-like (non-browser) environment such as Node, you'll need to disable style loading.
-
-```typescript
-import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
-initializeIcons('dist/');
-
-// Configure load-themed-styles to avoid registering styles.
-let themeLoader = require('@microsoft/load-themed-styles');
-themeLoader.configureLoadStyles(styles => {
-  // noop
-});
-
-// Set ssr mode to true, and rtl to false.
-let library = require('office-ui-fabric-react/lib/Utilities');
-library.setSSR(true);
-library.setRTL(false);
-
-// Assume a large screen.
-let responsiveLib = require('office-ui-fabric-react/lib/utilities/decorators/withResponsiveMode');
-responsiveLib.setResponsiveMode(responsiveLib.ResponsiveMode.large);
-```
-
-You'll also want to mock out requiring `.scss` files.
-In Jest:
+### Server-side rendering
 
-```js
-  moduleNameMapper: {
-    // jest-style-mock.js should just contain module.exports = {};
-    '\\.(scss)$': path.resolve(__dirname, 'jest-style-mock.js'),
-  }
-```
+Fabric components can be rendered in a server-side Node environment (or used in tests which run in an SSR-like environment), but it requires customizing how styles and SCSS files are loaded. See the [server-side rendering documentation](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Server-side-rendering-and-browserless-testing) for examples of how to handle this.
 
-## Advanced usage
+### Advanced usage
 
-For advanced usage including info about module vs. path-based imports, using an AMD bundler like Require, and deployment features, see our [advanced documentation](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Advanced-Usage).
+For info about advanced usage including module- vs. path-based imports, using an AMD bundler like RequireJS, and deployment features, see our [advanced documentation](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Advanced-Usage).

```

Comment:
Intentional hyphen?

Reply:
Yes. Not sure of the official grammatical term for this, but since both "module" and "path" relate to "based" (and you'd say "module-based" or "path-based" if they were together), I'm pretty sure "module" should also have a hyphen after.